backend/src/test/resources/application.properties

```diff
@@ -1 +1,3 @@
-spring.mongodb.embedded.version=6.0.1
\ No newline at end of file
+spring.mongodb.embedded.version=6.0.1
+product.details=/Users/kareem89/IdeaProjects/simple-onlineshope-my-capstone-project/backend/product_details.txt
```

Comment:
das müsste dir eigentlich in den tests in den github actions irgendwie um die ohren fliegen, weil dort das file unter dem pfad nicht existiert

Reply:
danke ich probiere ohne 
ich hab das wegen gemacht weil alle tests sind ausgefallen